src/ert/storage/local_storage.py

```diff
@@ -212,6 +212,13 @@ def __init__(
 
                     observations.migrate(self.path)
                     self._add_migration_information(3, "observations")
+                elif version == 4:
+                    from ert.storage.migration import (
+                        experiment_id,
+                    )
+
+                    experiment_id.migrate(self.path)
+                    self._add_migration_information(4, "experiment_id")
```

Comment:
This will have to be added to the other versions as well. The reason is because we always jump straight to the latest version, and dont upgrade iteratively. It might be time to start doing that, but that is outside the scope here.